### ISSUE TITLE
About ver2 in code

### ISSUE THREAD

**Issue by isht7** [brade31919/radar_depth#4]
Thank you very much for releasing the code. 
I noted in the processed data that I downloaded, there were 2 folders, `ver2_lidar1_radar1` and `ver2_lidar1_radar3_radar_only`. It seems to me that `ver2_lidar1_radar1` contains image, lidar and some radar information(probably front radar?), and `ver2_lidar1_radar1` contains another set of radar information (back radar)? It would be great if you can tell me more about this.

It seems to me that when "ver3" is being used, radar data in folder `ver2_lidar1_radar3_radar_only` is used instead of the other folder as seen [here(keys like 'radar_points' are overwritten?)](https://github.com/brade31919/radar_depth/blob/6dc235433510eb7e24195a55565ea9d906e7c0d3/dataset/nuscenes_dataset_torch_new.py#L634-L635).

I wanted to train a model which can do depth prediction on all the 6 cameras, in addition to the front and back that are done now. It seems to me that 'ver2' of the code might do something like that. I noted that `ver2` of the code uses only 1 folder to train - `ver2_lidar1_radar1`. When I changed `version = "ver2"` in the config file, I encountered in an error - 

![image](https://user-images.githubusercontent.com/21278652/106912767-312d4c80-66b8-11eb-8975-bd0f84c06d85.png)

(I updated the file `nuscenes_day_night_info.pkl` before running `ver2`)
Would you know how I could fix this error? I wanted to train a model on all the 6 cameras of the dataset. If you could give some hints on how I could do this, that would be very helpful. Thank you very much for your time!

**Comment by isht7** [comment]
Upon further inspection it seems that the error occurs when the there are too few points(1 or 0) in `input_data['radar_points']`. Could you suggest a fix to this?

**Comment by brade31919** [comment]
Hi @isht7 

The ver2_lidar1_radar3_radar_only is more like a patch for more radar information. In ver3, if I remember correctly, I used only the front and back camera + Radar points, but we accumulate the radar points along the temporal axis a bit to get more points.

Regarding the ver2 issue, as I indicated in README.md, I haven't finished the documentations on dataset yet. The processed dataset was originally extracted from the nuScenes database. The code used to export them can be found [here](https://github.com/brade31919/radar_depth/blob/master/dataset/nuscenes_export.py). However, I haven't tested them before the release of the code (I don't have the machine and storage to do that now since students need to leave the lab after the project). The top priority in this repo is that the users can used the training code and processed dataset to get the similar results I got on the paper.

Regarding the projection of radar points onto other camera, you can check the code [here](https://github.com/brade31919/radar_depth/blob/master/dataset/nuscenes_dataset.py) and try it yourself. I remember that you can't get like more than 10 points which is quite frustrating. That's the reason why I changed back to front + back views.
